pkgs/development/libraries/mesa/default.nix

```diff
@@ -51,9 +51,9 @@
       "imagination-experimental" # PowerVR Rogue (currently N/A)
       "panfrost" # ARM Mali Midgard and up (T/G series)
     ]
-    ++ lib.optionals stdenv.isx86_64 [
-      "intel" # Intel (aka ANV), could work on non-x86_64 with PCIe cards, but doesn't build as of 22.3.4
```

Comment:
This comment should probably be updated to "x86" instead of removed, because it still doesn't build on aarch64 AFAICT.

Reply:
Done.